sammi/cdf_attribute_manager.py

```diff
@@ -312,3 +538,89 @@ def get_variable_attributes(
                 output[attr_name] = ""
 
         return output
+
+    def variable_attribute_template(self) -> OrderedDict:
```

Comment:
You don't need to specifically use OrderedDict post Python 3.7, as dicts by default keep the order of keys.